figshare/figshare.py

```diff
@@ -290,3 +293,37 @@ def get_file_details(self, article_id, file_id):
         response = issue_request('GET', url,
                                  headers=self.get_headers(token=self.token))
         return response
+
+    def retrieve_files_from_article(self, article_id, outpath=''):
+        """ Retrieve files and save them locally.
+
+        By default, files will be stored in the current working directory
+        under a folder called figshare_<article_id> by default.
+        Specify <outpath> for: <outpath>/figshare_<article_id>
+
+        Parameters
+        ----------
+        article_id : str or int
+            Figshare article ID
+
+        """
+
+        # Handles missing trailing forward slash
+        if outpath != '':
+            if outpath[-1] != '/':
+                outpath += '/'
```

Comment:
use `os.path.join` to combine file path and get rid of this.

Reply:
Done.